doc/ref/chap-type-method.md

```diff
@@ -641,6 +641,20 @@ TODO:
 call mydict->accum('key', 'string to append')
 ```
 
+### inc()
+
+Increments the value by the increment specified. If there is no value in the
+dictionary with the provided key, its default value is 0 of the type of the
+increment (int or float). If the increment is not specified, its default value
+is 1 of the type of the value with the provided key. If neither the increment
```

Comment:
I think this is better illustrated by examples that the user can run

I know we are not consistent with that yet, but we should move toward it

And actually I started it with

    build/doc.sh run-code-all

That actually runs the code examples in `chap-type-method` that are NOT `raw`

So we should change these to 4 space indents, which means they are runnable code blocks

And then illustrate the int and float cases separately, what happens when the arg is missing, when the key is missing, etc.